tiny_tokenizer/tiny_tokenizer_token.py

```diff
@@ -25,27 +28,38 @@ def __init__(
             surface (original form) of a word
         postag (str, default: None)
             part-of-speech tag of a word (optional)
+        postag1 (str, default: None)
+            detailed part-of-speech tag of a word (optional)
         postag2 (str, default: None)
             detailed part-of-speech tag of a word (optional)
-        conjugate type (str, default: None)
+        postag3 (str, default: None)
+            detailed part-of-speech tag of a word (optional)
+        inflection (str, default: None)
             conjugate type of word (optional)
-        conjugate form (str, default: None)
+        conjugation (str, default: None)
             conjugate type of word (optional)
         origin_form (str, default: None)
             original form of a word
         yomi (str, default: None)
             yomi of a word (optional)
         pron (str, default: None)
             pronounciation of a word (optional)
+        normalized_form (str, default: None)
+            normalized form of a word (optional)
+            Note that normalized_form is only
+            available on SudachiPy
         """
         self.surface = surface
         self.postag = postag
+        self.postag1 = postag1
         self.postag2 = postag2
-        self.conj_type = conj_type
-        self.conj_form = conj_form
+        self.postag3 = postag3
+        self.inflection = inflection
+        self.conjugation = conjugation
         self.origin_form = origin_form
         self.pron = pron
         self.yomi = yomi
```

Comment:
https://github.com/himkt/tiny_tokenizer/pull/37/files#diff-65a98114777daebf655054b1de7988a4L36-L38
`pron` and `yomi` is up-side-down compare with above link.

Reply:
You're right